tests/scripts/task_microtvm_cpp_tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
is this used anywhere?

Reply:
ah forgot to add calls to these in the relevant jobs after their C++ testing, fixed